sceptre/scipool/sc-tag-options/internal/Departments.json

```diff
@@ -1,6 +1,7 @@
 [
-  "CompOnc",
+  "SCCE",
```

Comment:
please put in alphabetical order.  also could you please write a more descriptive message about this tag in the PR description?

Reply:
CompOnc is now renamed to SCCE which is short for: Science Coordination & Community Engagement. CNB is short for Challenges & Benchmarking.

You can see our current org here: https://sites.google.com/sagebase.org/intranet/team-sage/directory-org-chart/